test/cql-pytest/test_aggregate.py

```diff
@@ -76,13 +76,14 @@ def test_count_in_partition(cql, table1):
 # value in v
 def test_count_specific_column(cql, table1):
     p = unique_key_int()
-    stmt = cql.prepare(f"insert into {table1} (p, c, v) values (?, ?, ?)")
-    cql.execute(stmt, [p, 1, 1])
-    cql.execute(stmt, [p, 2, 2])
-    cql.execute(stmt, [p, 3, 3])
-    cql.execute(stmt, [p, 4, None])
+    stmt = cql.prepare(f"insert into {table1} (p, c, v, fli) values (?, ?, ?, ?)")
+    cql.execute(stmt, [p, 1, 1, [1]])
+    cql.execute(stmt, [p, 2, 2, None])
+    cql.execute(stmt, [p, 3, 3, [2, 3]])
+    cql.execute(stmt, [p, 4, None, None])
```

Comment:
Nice. If I understand correctly, your patch also changes (fixes) the behavior of counting tuples and UDTs, so would have been nice to also test those, I think those would also fail before your patch and succeed before it?

By the way, please check that the test works on Cassandra (`test/cql-pytest/run-cassandra test_aggregate.py::test_count_specific_column`).

Reply:
ok, will do both

Reply:
Is there a way to create and use a UDT fixure?

Reply:
I don't think we have an example of a UDT fixture (I didn't write many UDT tests myself), but we do have a `new_type()` context manager for creating the type and `user_type()` function to create an object to use in a parepared statement. You can see examples in test_filtering.py.